aspnet/getting-started.rst

```diff
@@ -0,0 +1,53 @@
+Getting Started
+===============
+
+1. Install `.NET Core`_
+
+2. Create a new .NET Core project:
+
+  .. code-block:: console
+    
+    mkdir aspnetcoreapp
+    cd aspnetcoreapp
+    dotnet new
+
+3. Update your ``project.json`` file to add the package for the Kestrel HTTP server and the ASP.NET Core hosting model:
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/project.json
+    :language: c#
+    :emphasize-lines: 11
+
+4. Restore the packages for your project:
+
+  .. code-block:: console
+    
+    dotnet restore
+
+5. Add a ``Startup.cs`` file that defines the request handling logic your app:
```

Comment:
logic **FOR** your app:
My pref would be:
Add a ``Startup.cs`` file that defines the request handling logic: